MAINTAINERS

```diff
@@ -0,0 +1,12 @@
+If you'd like to make a change to this library, you should contact at least one
+of the maintainers listed below. They will be able to discuss the change with
+you and decide the next best step. This maintainers file should be kept up to
+date with the current maintainers of the project.
```

Comment:
Add "The maintainers will also be notified of pull requests as described in the README." after "the next best step."